python_examples/age_stratified_example/age_stratified_plot.py

```diff
@@ -0,0 +1,239 @@
+        infection_category : str
+            Category to be plotted, defaults to Total Infectious
+        write_Df_toFile : str
+            Optional argument, .csv filepath if configured plotting
+            data should be written to file
+        param_file : str
+            Parameters needed to perform latent time convolution
+        """
+        if not param_file:
+            print('Need parameters for weekly convolution')
+        else:
+            self._convolveLatentTime(param_file)
+        if infection_category == "Total Infectious":
+            self._sum_infectious()
+        if self.do_ages and self.age_list[0] == '0-5':
+            # If data is presented in 5 year age groups.
+            self._5yrAgeGroupsTo10()
+        new_frame = self.data.loc[:, ('time', infection_category)]
+        time_col = 'time'
+        if self.start_date is not None:
+            # If a start date is given, plot with dates on x axis.
+            time_col = 'dates'
+            if self.sum_weekly:
+                # By giving the same 'dates' for each weekday, the
+                # 7 day total is automatically found.
+                new_frame['dates'] = self._dates(new_frame, 'weekly')
+            else:
+                new_frame['dates'] = self._dates(new_frame, 'daily')
+
+        if self.do_ages:
+            # If we have age stratified data, plot the bar chart with
+            # colours for each age.
+            new_frame.loc[:, self.age_name] = self.data.loc[:, self.age_name]
+            new_frame = new_frame.groupby([time_col, self.age_name]) \
+                .sum().reset_index()
+            new_frame = new_frame.pivot(index=time_col, columns=self.age_name,
+                                        values=infection_category)
+            if self.age_list:
+                # Renames columns to actual age ranges if given.
+                idx = list(new_frame.columns)[0]
+                for i in range(len(list(new_frame.columns))):
+                    new_frame = new_frame \
+                                    .rename(columns={i+idx: self.age_list[i]})
+            if write_Df_toFile is not None:
+                new_frame.to_csv(write_Df_toFile)
+            new_frame.plot.bar(stacked=True, colormap="inferno_r")
+        else:
+            new_frame = new_frame.groupby([time_col]) \
+                .sum().reset_index()
+            new_frame.plot.bar(x=time_col, y=infection_category)
+        if self.sum_weekly:
+            title = "Weekly cases by age"
+        else:
+            title = "Daily cases by age"
+        if param_file:
+            title = 'New ' + title
+        plt.title(title)
+        plt.gca().legend().set_title('')
+        plt.xlabel("Time")
+        plt.savefig(outfile)
+
+
+if __name__ == '__main__':
+    dirname = os.path.dirname(os.path.abspath(__file__))
+    p = Plotter(os.path.join(os.path.dirname(__file__),
+                "simulation_outputs/output_withage.csv"),
```

Comment:
```suggestion
                "simulation_outputs/output_with_age.csv"),
```
&c
Its not that important, its just withage started to look [funny](https://en.wiktionary.org/wiki/-age#English)